bundles/mapping/mapmodule/mapmodule.ol3.js

```diff
@@ -104,6 +104,9 @@ Oskari.clazz.define('Oskari.mapframework.ui.module.common.MapModule',
             });
 
             map.on('singleclick', function (evt) {
+                if (me.getDrawingMode()) {
+                    return;
```

Comment:
The same handling should be added to ol2 mapmodule to make the functionality consistent.